k8s/k8s-README.md

```diff
@@ -0,0 +1,194 @@
+# Kubernetes w/Contrast Java agent Example
+
+This guide provides a working example of setting up and configuring the Contrast Java agent within a kubernetes environment.   This guide assumes you have basic working knowledge of git, docker and kubernetes.
+
+## Prerequisites
+
+Setup a local kubernetes environment if you do not already have one available to you.  Below are a few of the available options.
+
+- Install Docker & kubernetes, follow instructions on [https://docs.docker.com/docker-for-mac/install](https://docs.docker.com/docker-for-mac/install/)/
+- Install and start Minikube, follow instructions on https://kubernetes.io/docs/tasks/tools/install-minikube/
+
+Clone the following repo that will be used for this tutorial:
+
+```shell
+git clone https://github.com/Contrast-Security-OSS/spring-petclinic.git
```

Comment:
Should this be https://github.com/Contrast-Security-OSS/demo-petclinic.git?

Reply:
Whoops, yup just fixed it